go.mod

```diff
@@ -1,86 +1,168 @@
 module github.com/GlintPay/gccs
 
-go 1.22.0
-
-toolchain go1.23.2
+go 1.23.6
 
 require (
+	codnect.io/chrono v1.1.3
 	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/caarlos0/env/v6 v6.10.1
 	github.com/emirpasic/gods v1.18.1
-	github.com/go-chi/chi/v5 v5.1.0
+	github.com/getsops/sops/v3 v3.9.4
+	github.com/go-chi/chi/v5 v5.2.1
```

Comment:
Personally, I'd have pushed out some of the unrelated dep updates separately to thin down the PR, but it's not a problem.